src/shared/discovery.js

```diff
@@ -33,15 +33,17 @@
 export default class Discovery {
   /**
    * @param {Window} target
-   * @param {Object} options
+   * @param {object} [options]
+   * @param {boolean} [options.server]
+   * @param {string} [options.origin]
    */
   constructor(target, options = {}) {
     /** The window to send and listen for messages with. */
     this.target = target;
 
     /**
      * Set whether this frame acts as a server (fetches annotations from the
-     * API) or a client (contains annotatable content and displays highlights).
+     * API) or a client (contains annotable content and displays highlights).
```

Comment:
The previous spelling was actually correct here. See https://en.wiktionary.org/wiki/annotatable.

Reply:
Corrected